sdk/src/main/java/io/opentelemetry/sdk/trace/export/package-info.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Utilities that allows different tracing services to export recorded data for sampled spans in
+ * their own format.
+ *
+ * <h2>Contents</h2>
+ *
+ * <ul>
+ *   <li>{@link io.opentelemetry.sdk.trace.export.SpanExporter}
+ *   <li>{@link io.opentelemetry.sdk.trace.export.SimpleSpansProcessor}
+ *   <li>{@link io.opentelemetry.sdk.trace.export.BatchSpansProcessor}
+ *   <li>{@link io.opentelemetry.sdk.trace.export.MultiSpanExporter}
+ * </ul>
+ *
+ * <h2>Default values for {@link io.opentelemetry.sdk.trace.export.BatchSpansProcessor}</h2>
```

Comment:
This document should specify both the env vars, the system properties, and the defaults, I think.